ChangeLog

```diff
@@ -1,8 +1,9 @@
 
 ### 3.8.0rc1dev <- NOTE: the release version number will be 3.8.0 ###
 
-- Mac: Updated to use Qt 5.12.10 for better compatibility with Big Sur (#1687).
-  This drops support for Yosemite and El Capitan in the distributed version.
+- Mac: Generate build with Qt 5.15.2 for better compatibility with Big Sur (#1687, #1768).
+  We still build Jamulus (legacy) with Qt 5.9.9 to support older versions of macOS; However
+  Big Sur users should use the standard build.
```

Comment:
We should specify that the standard build should be used by everyone running macOS 10.13 (High Sierra) or newer, and that the legacy build should _only_ be used on Yosemite, El Capitan or Sierra.